pkg/kubelet/cm/memorymanager/policy_static.go

```diff
@@ -327,7 +340,7 @@ func getPodRequestedResources(pod *v1.Pod) (map[v1.ResourceName]uint64, error) {
 	// Total resources requested by restartable init containers.
 	reqRsrcsByRestartableInitCtrs := make(map[v1.ResourceName]uint64)
 	for _, ctr := range pod.Spec.InitContainers {
-		reqRsrcs, err := getRequestedResources(pod, &ctr)
+		reqRsrcs, _, err := getRequestedResources(pod, &ctr)
```

Comment:
It's odd to me to have a function that "most of the time" has one of its return values ignored. It feels like there should be a better abstraction here.

Reply:
good point. I need the extra return value only for better error message, to express memory in terms of human friendl units (e.g. `1000Mi`) rather than in bytes. I'd think about something nicer.

Reply:
I _think_ I found a better way given how we want to use the data